### ISSUE TITLE
[devcontainers] Add useful components to Styles Devcontainer

### ISSUE THREAD

**Issue by wilkus27** [openstax/cnx-recipes#5157]
Add `liquidprompt` to Styles Devcontainer. It's used in Cookbook.

More information: https://liquidprompt.readthedocs.io/en/latest/overview.html

**Comment by LostMojo** [comment]
Stylelint investigation:

- Current installed version is 14.5, most recent version on their site is 15.0.0, which deprecates most of the style linting (i.e. quotation marks, indentation)
  - If we want 15.0.0 AND style correction, we could try adding [Prettier](https://prettier.io/).

- Main useful non-style features are currently disallowing px measurements when rem measurements can be used
- It should also catch things like too-long lines, Non-camelCased functions. 
- It would take some work, but depending on how important some of these rules are, and how much we expect for new people to be making additions, we could add a github test for it, like rubocop for cookboook
 - Specifically, we'd have to get rid of or change several of the standard rules, like the at-partial-leading-underscore or color hex case
  - or tweak our code to follow them (Can be done piece by piece, only running checks on files that have been changed in the PR-- should also cut down on testing time )

It's enough that I'm going to split it off into it's own separate card, away from liquid prompt!

**Comment by LostMojo** [comment]
Recommendation: If we're going to be sticking with sass for the long term, buffing up the scss linter makes sense. If we're switching over to typescript, XML, TSX, etc. then we'd probably be looking at just a css linter for the css they produce instead-- this could be useful for making sure we avoid mistakes like double styling a selector-- but what we'd need to look out for would change depending on the quirks of whatever css generation method we end up using.

TL;DR: delay until Final Prototype for stylish is determined, or needed for CSS or SCSS validation (Ex: if a LOT of non-CE-Styles folks are making PRs), otherwise, we'll need to redo a lot of work.
